ledger/participant-integration-api/src/main/scala/platform/store/backend/common/DebugStorageBackendTemplate.scala

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend.common
+
+import anorm.{RowParser, SQL}
+
+import java.sql.Connection
+import anorm.SqlParser.long
+import anorm.~
+import com.daml.platform.store.backend.DebugStorageBackend
+
+private[backend] trait DebugStorageBackendTemplate extends DebugStorageBackend {
```

Comment:
I agree that the naming is strange. I have chosen the name because of consistency: all similar files are called `*StorageBackendTemplate`, even though half of them don't use any strategy or template methods. Let's discuss the file naming separately.